tests/docker/Dockerfile

```diff
@@ -149,7 +149,7 @@ RUN useradd -u $UID -ms /bin/bash ducker \
   && mkdir -p /home/ducker/ \
   && rsync -aiq /root/.ssh/ /home/ducker/.ssh \
   && chown -R ducker /home/ducker/ /mnt/ /var/log/ \
-  && echo "PATH=$(runuser -l ducker -c 'echo $PATH'):$JAVA_HOME/bin" >> /home/ducker/.ssh/environment \
+  && echo "PATH=$(runuser -l ducker -c 'echo $PATH'):$JAVA_HOME/bin" >> /etc/environment \
```

Comment:
It seems the user enviroment file has not been applied. The new image sets the link to `/usr/bin/java`, which is why it works even though the environment file is not applied

Reply:
Yeah, thats correct, and I manually set soft link for Java tools to make it work, see description for more information.